laspy/base.py

```diff
@@ -159,6 +160,42 @@ def point_map(self):
         self.pointfmt = np.dtype([("point", [(str(x.name), x.np_fmt) for x in self.manager.point_format.specs])])
 
         if self.manager.header.version not in ("1.3", "1.4"):
+            if self.compressed:
+                import lazperf
```

Comment:
Shouldn't you check the status of `HAVE_LAZPERF` before moving on here?

Reply:
doh